Classes/EventListener/EventListenerInvoker.php

```diff
@@ -26,27 +26,38 @@
 /**
  * @Flow\Scope("singleton")
  */
-final class EventListenerInvoker
+class EventListenerInvoker
```

Comment:
As discussed: Instead of removing the `final` we should introduce a strategy to replace the `getEventStoreForEventListener()` method